server/internal/adapter/gql/gqlmodel/convert.go

```diff
@@ -14,38 +16,37 @@ func RefToString(s *string) string {
 	return *s
 }
 
-//func stringToRef(s string) *string {
-//	if s == "" {
-//		return nil
-//	}
-//	return &s
-//}
+// func stringToRef(s string) *string {
+// 	if s == "" {
+// 		return nil
+// 	}
+// 	return &s
+// }
 
 func BoolToRef(b bool) *bool {
 	return &b
 }
 
-//func ToPageInfo(p *usecase.PageInfo) *PageInfo {
-//	if p == nil {
-//		return &PageInfo{}
-//	}
-//	return &PageInfo{
-//		StartCursor:     p.StartCursor(),
-//		EndCursor:       p.EndCursor(),
-//		HasNextPage:     p.HasNextPage(),
-//		HasPreviousPage: p.HasPreviousPage(),
-//	}
-//}
+func ToPageInfo(p *usecase.PageInfo) *PageInfo {
+	if p == nil {
+		return &PageInfo{}
+	}
+	return &PageInfo{
+		StartCursor:     p.StartCursor(),
+		EndCursor:       p.EndCursor(),
+		HasNextPage:     p.HasNextPage(),
+		HasPreviousPage: p.HasPreviousPage(),
+	}
+}
 
-//
-//func ToPagination(pagination *Pagination) *usecase.Pagination {
-//	if pagination == nil {
-//		return nil
-//	}
-//	return &usecase.Pagination{
-//		Before: pagination.Before,
-//		After:  pagination.After,
-//		First:  pagination.First,
-//		Last:   pagination.Last,
-//	}
-//}
+// func ToPagination(pagination *Pagination) *usecase.Pagination {
+// 	if pagination == nil {
+// 		return nil
+// 	}
+// 	return &usecase.Pagination{
+// 		Before: pagination.Before,
+// 		After:  pagination.After,
+// 		First:  pagination.First,
+// 		Last:   pagination.Last,
+// 	}
+// }
```

Comment:
```suggestion
func ToPagination(pagination *Pagination) *usecase.Pagination {
	if pagination == nil {
		return nil
	}
	return &usecase.Pagination{
		Before: pagination.Before,
		After:  pagination.After,
		First:  pagination.First,
		Last:   pagination.Last,
	}
}
```

Public methods may be uncommented out, as they do not generate warnings even if they are not in use.